docs/fast_data/configuration/single_view_trigger_generator.md

```diff
@@ -40,12 +40,38 @@ When creating the service from the marketplace the following environment variabl
 | MANUAL_STRATEGIES_FOLDER         | -        | Path to the custom strategies folder where the custom strategies scripts are stored                                                                                                                     | -             |
 | TRIGGER_CUSTOM_FUNCTIONS_FOLDER  | -        | Path to the custom functions folder used in `__fromFile__` values                                                                                                                                       | ''            |
 
-## Config Maps
+## Attaching a Service to a Single View
+
+:::info
+This feature is available from version `v.11.8.0` of Mia-Platform Console.
```

Comment:
@TheRusCompass safe to assume we won't have a `11.8.0`, correct?

Reply:
let's see what we will have, thanks Deme